www-apps/icingaweb2-module-incubator/icingaweb2-module-incubator-0.20.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Bleeding edge Icinga Web 2 libraries"
+HOMEPAGE="https://github.com/Icinga/icingaweb2-module-incubator/"
+KEYWORDS="~amd64 ~x86"
+SRC_URI="https://codeload.github.com/Icinga/${PN}/tar.gz/v${PV} -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="php_targets_php8-1"
+PHP_DEPEND="
+	php_targets_php8-1? ( dev-lang/php:8.1[curl] )
```

Comment:
ditto

Reply:
same goes for icingaweb2-module-incubator... not compatible with 8.2